nixos/modules/virtualisation/qemu-vm.nix

```diff
@@ -182,6 +182,7 @@ let
           -L ${nixStoreFilesystemLabel} \
           -U eb176051-bd15-49b7-9e6b-462e0b467019 \
           -T 0 \
```

Comment:
Notice that the builder sets `-T 0`, but afaik, the SOURCE_DATE_EPOCH env has precedence over -T, so the clamping will happen on 1980 instead 1970, and for builds that non-deterministically touch files in the nix sandbox, this will lead to reproducibility issues, too. Solution would be to set SDE to 0.

Reply:
Wouldn't we achieve the same if we just didn't use the `-T` flag and instead only relied on SOURCE_DATE_EPOCH?